realscale.py

```diff
@@ -37,10 +37,43 @@ def __init__(self):
         self.OptionParser.add_option("--unit", action="store", type="string", dest="unit", default="cm",
                                      help="Real-world unit")
 
+    def calc_scale_center(self, p1x, p1y, p2x, p2y):
+        """ Use straight line as scaling center.
+            - determine which point is center on basis of quadrant the line is in.
+            - approx this by using center of line
+            0,0 corresponds to UL corner of page
+        """
+        scale_center = [0,0] # resulting scaling point
+        # calc page center
+        pagecenter_x = self.unittouu(self.document.getroot().get('width'))/2
```

Comment:
@Neon22 : For compatibility with older versions, I'd prefer if the getUnittouu() method would be used instead of self.unittouu().